package.json

```diff
@@ -79,7 +79,7 @@
     "canvas-fit": "^1.5.0",
     "color-alpha": "1.0.4",
     "color-normalize": "1.5.0",
-    "color-parse": "1.3.8",
+    "color-parse": "2.0.0",
```

Comment:
Please revert this dependency change. Upgrades like this could be proposed in a separate PR.